examples/finance/bs_ivbp.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>any reason to use the low level <code>second_derivative</code>  function rather than the shortcuts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done, had to add side="centered" on v.ds.